lib/ansible/template/__init__.py

```diff
@@ -103,7 +103,7 @@ def generate_ansible_template_vars(path, fullpath=None, dest_path=None):
     managed_str = managed_default.format(
         host=temp_vars['template_host'],
         uid=temp_vars['template_uid'],
-        file=temp_vars['template_path'],
+        file=temp_vars['template_path'].replace('%', '%%'),
```

Comment:
This fix only addresses the %, it should escape any/all characters that would cause issues with templating

Reply:
`{{` .. strftime is not the only 'templating' that happens